src/include/daos_obj_class.h

```diff
@@ -142,29 +142,22 @@ enum {
 	 * SX : spreading across all targets within the pool
 	 */
```

Comment:
We probably need to do further cleanups here, since those pre-registered class will only be there as a convenience for the user to specify a class manually. Otherwise, the automatic selection (daos_obj_generate_oid()) will just generate the class on the fly (without using this array of pre-defined class).